hw/top_darjeeling/ip_autogen/otp_ctrl/dv/env/otp_ctrl_mem_bkdr_util.core

```diff
@@ -0,0 +1,25 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: lowrisc:opentitan:top_darjeeling_otp_ctrl_mem_bkdr_util:0.1
+description: "OTP_CTRL mem_bkdr_util support package"
+virtual:
+  - lowrisc:ip_interfaces:otp_ctrl_mem_bkdr_util
+
+filesets:
+  files_dv:
+    depend:
+      - lowrisc:dv:mem_bkdr_util
+      - lowrisc:dv:crypto_dpi_present
+      - lowrisc:dv:lc_ctrl_dv_utils
+      - lowrisc:opentitan:top_darjeeling_otp_ctrl_top_specific_pkg:1.0
```

Comment:
Question: now that we use a virtual core for this and that we have `top_darjeeling` in the core name, should we remove `_top_specific` from the name?

Reply:
This is a bit verbose but we need to make a distinction between the otp_ctrl_pkg, which is generic and can be used by any IP, and the package that holds top specific stuff. With the existing VLNV convention if we name this "specific_otp_ctrl_pkg", the resulting name would be "top_<foo>_specific_otp_ctrl_pkg" which avoids repeating top, but when we change our VLNV convention this would be ...:<topname>_dv:specific_otp_ctrl_pkg, which is vague. We need to describe "top_specific" or "non_generic".

This reminded me to create an issue for this: https://github.com/lowRISC/opentitan/issues/25921.

Reply:
Thanks for the explanation :-)